src/compiler/utils.jl

```diff
@@ -20,7 +20,11 @@ function get_function!(builderF, mod::LLVM.Module, name)
 end
 
 
-T_ppjlvalue(ctx) = LLVM.PointerType(LLVM.PointerType(LLVM.StructType(LLVMType[]; ctx)))
+function T_ppjlvalue(ctx)
```

Comment:
I think this makes sense to not take a context, and use the surrounding ctx

Reply:
I've moved the ctx wrapper to declare_pgcstack! otherwise there's no active context to T_ppjlvalue.